menu.py

```diff
@@ -870,22 +618,32 @@ def previous_menu_option(self,
         target_option = list(options_enum)[target_option_idx]
         self.change_menu_option(target_option)
 
-    def change_menu_option(self,
-                           target_option: SetupOption | WikiOption | MainMenuOption | VictoryType) -> None:
+    def change_menu_option(self, target_option: MenuOptions) -> None:
         """
         Select the given menu option.
         :param target_option: The target option to be selected.
         """
         # Based on the enum type of the target option, figure out which corresponding field we need to change.
         match target_option:
-            case SetupOption():
-                self.setup_option = target_option
-            case WikiOption():
-                self.wiki_option = target_option
-            case MainMenuOption():
-                self.main_menu_option = target_option
-            case VictoryType():
-                self.victory_type = target_option
+            case SetupOption(): self.setup_option = target_option
+            case WikiOption(): self.wiki_option = target_option
+            case MainMenuOption(): self.main_menu_option = target_option
+            case VictoryType(): self.victory_type = target_option
+
+    def get_option_colour(self, option: MenuOptions) -> pyxel.Colors:
```

Comment:
Couple of things with this one: if we're going to use that as the return type, I believe it has to be `pyxel.colors`. Also, semantically I'm not sure if that's what we really want, given that `pyxel.colors` is actually a list of integers, and we're returning just an integer here, not a list.